packages/web-app-files/src/components/Search/List.vue

```diff
@@ -1,52 +1,59 @@
 <template>
-  <div class="files-search-result">
-    <app-bar :has-bulk-actions="true" />
-    <no-content-message v-if="!paginatedResources.length" class="files-empty" icon="folder">
-      <template #message>
-        <p class="oc-text-muted">
-          <span v-if="!!$route.query.term" v-translate>No resource found</span>
-          <span v-else v-translate>No search term entered</span>
-        </p>
+  <div class="files-search-result oc-flex">
+    <files-view-wrapper>
+      <app-bar :has-bulk-actions="true" :side-bar-open="sideBarOpen" />
```

Comment:
```suggestion
      <app-bar :has-bulk-actions="false" :side-bar-open="sideBarOpen" />
```

We don't want to support bulk actions in the search result page at the moment. Please set to `false` :-)